Formula/execline.rb

```diff
@@ -17,7 +17,6 @@ class Execline < Formula
   def install
     system "./configure",
       "--prefix=#{prefix}",
-      "--disable-static",
```

Comment:
If upstream won't accept it it's a non-starter for us, unfortunately. I think we can leave `execline` as is but I'd rather we leave static linking disabled. Although it may be useful on its own I wonder if it's actually going to see widespread enough usage to leave it around.